cmd/utils/flags.go

```diff
@@ -805,51 +795,34 @@ func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
 		urls = params.RinkebyBootnodes
 	case ctx.GlobalBool(GoerliFlag.Name):
 		urls = params.GoerliBootnodes
-	case cfg.BootstrapNodes != nil:
-		return // already set, don't apply defaults.
 	}
 
-	cfg.BootstrapNodes = make([]*enode.Node, 0, len(urls))
-	for _, url := range urls {
-		if url != "" {
-			node, err := enode.Parse(enode.ValidSchemes, url)
-			if err != nil {
-				log.Crit("Bootstrap URL invalid", "enode", url, "err", err)
-				continue
+	if cfg.BootstrapNodes == nil {
+		cfg.BootstrapNodes = make([]*enode.Node, 0, len(urls))
+		for _, url := range urls {
+			if url != "" {
+				node, err := enode.Parse(enode.ValidSchemes, url)
+				if err != nil {
+					log.Crit("Bootstrap URL invalid", "enode", url, "err", err)
+					continue
+				}
+				cfg.BootstrapNodes = append(cfg.BootstrapNodes, node)
 			}
-			cfg.BootstrapNodes = append(cfg.BootstrapNodes, node)
-		}
-	}
-}
-
-// setBootstrapNodesV5 creates a list of bootstrap nodes from the command line
-// flags, reverting to pre-configured ones if none have been specified.
-func setBootstrapNodesV5(ctx *cli.Context, cfg *p2p.Config) {
-	urls := params.DiscoveryV5Bootnodes
-	switch {
-	case ctx.GlobalIsSet(BootnodesFlag.Name) || ctx.GlobalIsSet(BootnodesV5Flag.Name):
-		if ctx.GlobalIsSet(BootnodesV5Flag.Name) {
-			urls = splitAndTrim(ctx.GlobalString(BootnodesV5Flag.Name))
-		} else {
-			urls = splitAndTrim(ctx.GlobalString(BootnodesFlag.Name))
 		}
-	case ctx.GlobalBool(RinkebyFlag.Name):
-		urls = params.RinkebyBootnodes
-	case ctx.GlobalBool(GoerliFlag.Name):
-		urls = params.GoerliBootnodes
-	case cfg.BootstrapNodesV5 != nil:
```

Comment:
What happened to all these deleted cases? Previously we used the default bootnodes for mainnet and rinkeby and goerli (ropsten missing, fell back to mainnet v5). In the current code, if the user doesn't supply any v5 bootnodes, we'll leave them empty, making the light client non-functional.